app/models/katello/host/content_facet.rb

```diff
@@ -18,6 +18,8 @@ class ContentFacet < Katello::Model
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      attr_accessible :content_view, :content_view_id, :lifecycle_environment, :lifecycle_environment_id, :host
```

Comment:
You would think so, but if I remove that [this test](https://github.com/Katello/katello/pull/5753/files#diff-bed4686568d79cce918e7c5566807cd5R18) fails because it uses :content_view and :lifecycle_environment as params. I can change the test, but I think we would want to be able to create a  content_facet by passing the whole content_view or lifecycle_environment